website/src/siteConfig.tsx

```diff
@@ -1,7 +1,7 @@
 // List of projects/orgs using your project for the users page.
 
 export const siteConfig = {
-  editUrl: 'https://github.com/formik/formik/edit/master/docs/src/pages',
+  editUrl: 'https://github.com/formik/formik/edit/master/docs',
```

Comment:
Correction, the url should just be `.../edit/master` as the route provides `/doc/*`
```suggestion
  editUrl: 'https://github.com/formik/formik/edit/master',
```